src/bo4e/enum/strenum.py

```diff
@@ -12,3 +12,25 @@ class StrEnum(str, Enum):
     """
 
     # see https://docs.python.org/3/library/enum.html?highlight=strenum#others
+
+
+# pylint: disable=too-few-public-methods
+class DocumentedStrEnum(StrEnum):
+    """
+    A StrEnum that has docstrings attached to its members.
+    Use it like this:
+    class MyDocumentedEnum(DocumentedStrEnum):
+        Foo = "Serialized Foo", "my good docstring of Foo"
+        Bar = "Serialized Bar", "bar is not foo (this is a docstring)"
+    Then wherever MyDocumentedEnum is used in BOs or COMs, the attribute will serialize as either "Serialized Foo"
+    or "Serialized Bar". And inspect.getdocs(MyDocumentedEnum.Foo) will return the respective docstring.
+    This is unittested.
+    """
+
+    # see https://stackoverflow.com/a/50473952/10009545
+    def __new__(cls, value, doc=None):
+        self = str.__new__(cls, value)  # calling super().__new__(value) here would fail
```

Comment:
I tried it with `StrEnum` but it did not work =(
It seems that it can not find the right ENUM if we use the StrEnum class.

Reply:
same for me